features/caseworker_claims_list.feature

```diff
@@ -40,7 +41,7 @@ Feature: Caseworker claims list
      Then I should only see <archive_number> claims
       And the claim count should show <archive_number>
 
-     Examples:
+     Examples: List of defendants and the number of live/archive claims they're involved in
```

Comment:
I'm aware of points 1 and 2 but given the number of tests and time we have available I think updating the locations/tests in a piecemeal fashion is the only real option of getting it into a usable state.